dbux-code/src/chapterListBuilderView/chapterListBuilderViewController.js

```diff
@@ -127,9 +138,9 @@ export default class ChapterListBuilderViewController {
 
     progress?.report({ message: `Parsing application` });
     if (allApplications.selection.count !== 1) {
-      this.treeNodeProvider.warn(`Ran test, but found more than one application (selecting first).`);
+      warn(`Ran test, but found more than one application (selecting first).`);
     }
-    const ddgs = findDDGContextIdInApp(app, exercise);
+    const ddgs = this.findDDGContextIdInApp(app, exercise);
```

Comment:
fix name: function does not just find a `contextId`, but more than that. Maybe `findDDGInfo`?